### PR TITLE
added indent to expanded tabs, refactored code in nav-menu element

### PR DESCRIPTION
**Pull request by MickhaelSh**
## Description

Added disableGutters property to all list items, added paddings to list items and bage

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
|175270856-ceb32ae2-792e-4702-ae19-b410c01c362f |padding|

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅ All tests passed locally
- [x] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

**Comment by Pipuss** [comment]
Please, don't forget to check the report of statick analizer

File: src/components/nav-menu/nav-menu.js [hunk 1]
```diff
@@ -95,135 +88,99 @@ const NavMenu = ({ width }) => {
...
-  );
-
-  const staticPagesMenuItems = config.staticPagesCategories.map((category) =>
-    returnedList(category[0], category[1], category[2], classes.nested)
-  );
-
-  const constructorPagesMenuItems = config.constructorMenuCategories.map(
-    (category) =>
-      returnedList(category[0], category[1], category[2], classes.nested)
+  function makeMenuItems(arrayOfCategories, className) {
```
Comment (maarekua) [inline src/components/nav-menu/nav-menu.js:100]: Use the arrow function declaration, please.

File: src/components/nav-menu/nav-menu.js [hunk 2]
```diff
@@ -95,135 +88,99 @@ const NavMenu = ({ width }) => {
...
-    (category) =>
-      returnedList(category[0], category[1], category[2], classes.nested)
+  const certificatesMenuItems = makeMenuItems(
+    'certificatesMenuCategories',
+    'nested'
   );
 
-  const parentMenuTabsProperties = [
-    [
+  function createParentMenuTab(tabName, itemsInMenu, TAB, icon) {
```
Comment (maarekua) [inline src/components/nav-menu/nav-menu.js:121]: Same

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/components/nav-menu/nav-menu.js (hunk 1 above)
Comment (MickhaelSh) [reply src/components/nav-menu/nav-menu.js]: fixed

File: src/components/nav-menu/nav-menu.js (hunk 2 above)
Comment (MickhaelSh) [reply src/components/nav-menu/nav-menu.js]: fixed

**Review by IhorKytsak — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**